driver/GFDL/atmosphere.F90

```diff
@@ -385,29 +394,22 @@ subroutine atmosphere_init (Time_init, Time, Time_step, Surf_diff, Grid_box)
                                                       allocate(ttend(isc:iec, jsc:jec, 1:npz))
    if ( any((/ id_qdt_dyn, id_qldt_dyn, id_qidt_dyn, id_qadt_dyn /) > 0) .or. &
         query_cmip_diag_id(ID_tnhusa) .or. query_cmip_diag_id(ID_tnhus) )  allocate(qtend(isc:iec, jsc:jec, 1:npz, 4))
+
+! could zero out diagnostics if tracer field not defined
+   if (sphum > size(qtend,4)) id_qdt_dyn = 0
+   if (liq_wat > size(qtend,4)) id_qldt_dyn = 0
+   if (ice_wat > size(qtend,4)) id_qidt_dyn = 0
+   if (cld_amt > size(qtend,4)) id_qadt_dyn = 0
 !miz
 
-! get tracer number for common moisture tracers
-   nqv = get_tracer_index(MODEL_ATMOS,'sphum')
```

Comment:
I think the variables nqi, nqa, and nql are also in GFDL_tools/fv_cmip_diag.F90
For consistency, should these be also changed?

Reply:
This will get fixed first in master and then pushed to dev/gfdl to keep it in sync.

Reply:
Issue #20 has been created.